pipelines/rj_smtr/veiculo/flows.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf-8 -*-
+"""
+Flows for veiculos
+"""
+
+from prefect import Parameter, case
+from prefect.run_configs import KubernetesRun
+from prefect.storage import GCS
+
+# EMD Imports #
+
+from pipelines.constants import constants as emd_constants
+from pipelines.utils.decorators import Flow
+from pipelines.utils.execute_dbt_model.tasks import get_k8s_dbt_client
+from pipelines.utils.tasks import (
+    rename_current_flow_run_now_time,
+    get_current_flow_mode,
+    get_current_flow_labels,
+)
+
+# SMTR Imports #
+
+from pipelines.rj_smtr.constants import constants
+
+from pipelines.rj_smtr.schedules import (
+    every_day,
+)
+from pipelines.rj_smtr.tasks import (
+    create_date_hour_partition,
+    create_local_partition_path,
+    fetch_dataset_sha,
+    get_current_timestamp,
+    # get_local_dbt_client,
+    get_raw,
+    parse_timestamp_to_string,
+    save_raw_local,
+    save_treated_local,
+    set_last_run_timestamp,
+    upload_logs_to_bq,
+    bq_upload,
+)
+from pipelines.utils.execute_dbt_model.tasks import run_dbt_model
+
+from pipelines.rj_smtr.veiculo.tasks import (
+    pre_treatment_veiculos,
+)
+
+# Flows #
+
+with Flow(
+    "SMTR: Licenciamento de Veículos - Captura",
+    code_owners=["rodrigo", "fernanda"],
+) as vehicle_capture:
```

Comment:
```suggestion
) as veiculo_sppo_licenciamento_captura:
```
mesma coisa aqui, vamos adotar o pt-br inclusive para ficar igual em todos os lugares? (pipelines, dbt, BQ, etc)